itou/templates/apply/process_details_prescriber.html

```diff
@@ -52,8 +52,7 @@ <h3>Historique des modifications</h3>
     </div>
 
     <p class="text-right">
-        <a class="btn btn-link" href="{{ back_url }}">Retour</a>
-        <a class="btn btn-primary" href="{% url 'dashboard:index' %}">Fermer</a>
+        <a class="btn btn-primary" href="{{ back_url }}">Fermer</a>
```

Comment:
ce commit "Don't be stupid" (et probablement "Fix broken template" auparavant) pourraient etre squashés avec le commit qui les introduit ("Update job app details...")